### PR TITLE
[DOCS][8.2]Update Timeline UI images to include Session View

### LINKED ISSUE #1839: [DOCS] [8.2] Update Timelines topic
## Description 

Several updates to timelines are coming in 8.2. 

## Acceptance Test Criteria

Need new screenshots and updated text that illustrated the changes listed [here](https://docs.google.com/document/d/1STB8MrIsvtseOD20P1MC9NbI9u1moSup1v2bh3Pg4Qg/edit)

## Notes

Should be approved by Michael O.

### PR DESCRIPTION
**Pull request by benironside**
Fixes #1839 by updating the screenshots in the Timelines overview topic to include the Session View tab and icon. 

Preview: [Investigate events in Timeline](https://security-docs_1844.docs-preview.app.elstc.co/guide/en/security/master/timelines-ui.html)

### CODE REVIEW

File: docs/events/timeline-ui-overview.asciidoc [hunk 1]
```diff
@@ -30,18 +30,23 @@ retrieved from the alert. For more information, refer to <<timeline-templates-ui
...
+Using the drop-down options by the KQL bar, you can select whether <<det-engine-terminology, alerts>>,
+other raw events, or both are displayed in the Timeline.
 
 [discrete]
 [[conf-timeline-display]]
 == Configure Timeline event context and display
 
-Many types of events automatically appear in preconfigured views that provide relevant contextual information, called *Event Renderers*. You can see and turn them on or off with the Settings menu in the upper left corner of the results pane:
+Many types of events automatically appear in preconfigured views that provide relevant
+contextual information, called *Event Renderers*. You can see and turn them on or off
```
Comment (nastasha-solomon) [inline docs/events/timeline-ui-overview.asciidoc:41]:
```suggestion
contextual information, called *Event Renderers*. You can display and turn them on or off
```

**Review by nastasha-solomon — APPROVED** [review]
Just left one minor suggestion. Otherwise, LGTM thanks!

**Review by joepeeples — COMMENTED** [review]
Updated images LGTM!

In the diff there are several lines highlighted where it doesn't look like any changes have been made other than line breaks — these won't affect output but I wonder if they're necessary. I don't know if we have any guidelines on manually adding line breaks, but I generally don't do that because it introduces additional changes that Git will start tracking. 

@jmikell821, any context/guidelines on manual line breaks? Maybe ping the other writers?

**Comment by benironside** [comment]
> 

I just think it's a lot easier to read and edit asciidoc when it doesn't require left to right scrolling because the lines are shorter. I guess it's not necessary though because of the softwrap option. No strong feelings on it. Also curious to hear Janeen's thoughts.

> Updated images LGTM!
> 
> In the diff there are several lines highlighted where it doesn't look like any changes have been made other than line breaks — these won't affect output but I wonder if they're necessary. I don't know if we have any guidelines on manually adding line breaks, but I generally don't do that because it introduces additional changes that Git will start tracking.
> 
> @jmikell821, any context/guidelines on manual line breaks? Maybe ping the other writers?

**Comment by karanverma-qasource** [comment]
Hi @benironside,

We have reported the 01 issue for the docs related to this PR here:


- https://github.com/elastic/security-docs/issues/1877

We will validate this PR once the above issue is fixed.

Thanks!

**Comment by karanverma-qasource** [comment]
Hi @benironside,

We tested the linked docs for the reported bugs and found that issue is partially fixed. Please look into the same.

Thanks !

**Comment by karanverma-qasource** [comment]
Hi @benironside,

We tested the linked docs for the reported bugs and found that all the docs are correctly updated as per the latest UI. So we are good to go ahead and merge the changes.

Marking this ticket as 'QA validated'.

Thanks!

**Pull request merged**